listenstore/listenstore/listenstore.py

```diff
@@ -89,23 +87,28 @@ def insert_postgresql(self, listens):
                 if not listen.validate():
                     raise ValueError("Invalid listen: %s" % listen)
                 try:
-                    res = connection.execute(
-                    """ INSERT INTO listen(user_id, ts, artist_msid, album_msid, recording_msid, raw_data)
+                    connection.execute("""
+                        INSERT INTO listen(user_id, ts, artist_msid, album_msid, recording_msid, raw_data)
                         VALUES ( %(user_id)s, to_timestamp(%(ts)s), %(artist_msid)s, %(album_msid)s,
-                        %(recording_msid)s, %(raw_data)s) ON CONFLICT DO NOTHING """, self.format_dict(listen))
+                            %(recording_msid)s, %(raw_data)s)
+                        ON CONFLICT DO NOTHING
+                        """, self.format_dict(listen))
                 except Exception, e:     # Log errors
-                        self.log.error(e)
+                    self.log.error(e)
 
     def execute(self, query, params={}):
         with self.engine.connect() as connection:
             res = connection.execute(query, params)
             return res.fetchall()
 
     def fetch_listens_from_storage(self, user_id, from_id, to_id, limit, order, precision):
-        query = """ SELECT id, user_id, extract(epoch from ts), artist_msid, album_msid, recording_msid, raw_data """ + \
-                """ FROM listen WHERE user_id = %(user_id)s """ + \
-                """ AND extract(epoch from ts) > %(from_id)s AND extract(epoch from ts) < %(to_id)s  """ + \
-                """ ORDER BY extract(epoch from ts) """ + ORDER_TEXT[order] + """ LIMIT %(limit)s"""
+        query = """
+                SELECT id, user_id, extract(epoch from ts), artist_msid, album_msid, recording_msid, raw_data
```

Comment:
See the other comment I made about layout of queries. I think this was originally copied from the cassandra code, right? In that case it's OK that we've not formatted it yet, but I'd like to see all SQL statements uniformly formatted